src/main/kotlin/study/Skills.kt

```diff
@@ -0,0 +1,18 @@
+package study
+
+class Skills {
+    private val _skills: MutableList<Skill> = mutableListOf()
+
+    val skills: List<Skill>
+        get() {
+            return _skills.toList()
+        }
```

Comment:
방어적 복사 👍 

아래처럼 작성할수도 있겠네요~

```suggestion
        get() = _skills.toList()
```